integration/project.yaml

```diff
@@ -5,6 +5,7 @@ test:
     - "VAULT_TOKEN=mytoken"
     - "CIRCLE_SHA1=${CIRCLE_SHA1}"
     - "CIRCLECI=${CIRCLECI}"
+    - "CLUSTER_NAME=${CLUSTER_NAME}"
```

Comment:
I do not think this is missing. The tests write it in the go code now. Why should we add it?

Reply:
We talked about this here https://github.com/giantswarm/aws-operator/pull/831#discussion_r173858509 we need only to set it when it is not already set.

Reply:
But in which case does this fail now? Locally?

Reply:
Yes, locally it tries to set the cluster name based on `CIRCLE_SHA1`, which might be not set or set to a value that doesn't match with the chart pushed to quay.